orders/views.py

```diff
@@ -27,4 +27,24 @@ def get(self, request):
             return JsonResponse({'message':'KEY_ERROR'}, status=400)
         
         except User.DoesNotExist:
-            return JsonResponse({'message':'INVALID_USER'}, status=400)
\ No newline at end of file
+            return JsonResponse({'message':'INVALID_USER'}, status=400)
+
+    @login_decorator
+    def delete(self, request, cart_item):
+        try:
+            signed_user = request.user
+
+            if cart_item == 0:
+                items = CartItem.objects.filter(user=signed_user)
+                for item in items:
+                    item.delete()
```

Comment:
```suggestion
                iCartItem.objects.filter(user=signed_user).delete()
```
queryset도 delete() method가 적용됩니다.

Reply:
수정했습니다!